samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/WOCCatalog.vcxproj

```diff
@@ -274,6 +276,22 @@
       <VariableFile Condition="'$(Configuration)'=='Debug'">WOCCatalog-Debug-xcvars.txt</VariableFile>
       <VariableFile Condition="'$(Configuration)'=='Release'">WOCCatalog-Release-xcvars.txt</VariableFile>
     </SBInfoPlistCopy>
+    <SBResourceCopy Include="Base.lproj\Localizable.strings">
+      <FileType>Document</FileType>
+      <VariantDir>Base.lproj</VariantDir>
+    </SBResourceCopy>
+    <SBResourceCopy Include="de.lproj\Localizable.strings">
+      <FileType>Document</FileType>
+      <VariantDir>de.lproj</VariantDir>
+    </SBResourceCopy>
+    <SBResourceCopy Include="zh.lproj\Localizable.strings">
+      <FileType>Document</FileType>
+      <VariantDir>zh.lproj</VariantDir>
+    </SBResourceCopy>
+    <SBResourceCopy Include="zh-Hant-TW.lproj\Localizable.strings">
```

Comment:
is this the apple official name or the windows name for this locale?

Reply:
Windows.

Reply:
then you're not exercising the `_` variants